user-management/user-details/script.js

```diff
@@ -2,9 +2,13 @@ let userData = {};
 let userAllTasks = [];
 let userSkills = [];
 let userStatusData = {};
+let userAllPRs = [];
 let currentPageIndex = 1;
+let currentPRPageIndex = 1;
 let taskPerPage = 3;
 let totalPages = Math.ceil(userAllTasks.length / taskPerPage);
+let PRsPerPage = 3;
```

Comment:
All the constants should go inside constants file

Reply:
these are not **constants**.